wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Videos card

Fixes #10896 

This add the Period Videos card.
- Max 6 data rows.
- If more than 6, `View more` is added.
- If no data rows, `No data for this period` is displayed.
- Data rows:
  - All rows display the same video icon.
  - Selecting a row will display the media details (the same view that is displayed when selecting media from the Media Library).

To test:

---
- On a site with Video views, go to Stats > select a period.
- Verify the view is:
<img width="350" alt="videos" src="https://user-images.githubusercontent.com/1816888/51769573-af55c200-20a0-11e9-96e2-ff575c0f01dc.png">

---
- Click on a video row.
- On the media details, depending on if you are a site admin, you may see edit options.
  - Your permissions may differ, but I found that on en.blog I am not admin (and cannot edit media), and I am an admin on Hogwarts (thus I can edit media).

![media_details](https://user-images.githubusercontent.com/1816888/51770088-1162f700-20a2-11e9-99f5-2c93ac707b0a.png)

---
- On a site with no Videos views, verify the view is:
<img width="350" alt="nil" src="https://user-images.githubusercontent.com/1816888/51769500-74ec2500-20a0-11e9-81c5-0c9d289eb2bf.png">
